### PR TITLE
fix: balances table styles update

### PR DESCRIPTION
**Pull request by joanna-pagepro**
https://github.com/JoinColony/colonyCDapp/issues/1933

Updated mobile view:

- table styles
- spacing next to `Add funds to the Colony` removed
image

- meatball menu is full width now
image

### CODE REVIEW

**Review by warrencolliver — DISMISSED** [review]
Great work, looks so much better now.

**Review by ArmandoGraterol — CHANGES_REQUESTED** [review]
Good job! The only thing I could point out right now is that the locked icon seems to be really far away from the name of the token when it's long and goes to a new line when in the designs it is side by side:
Screenshot 2024-02-21 at 15 26 13

File: src/components/frame/v5/pages/BalancePage/partials/BalanceTable/hooks.tsx [hunk 1]
```diff
@@ -15,22 +20,70 @@ import TokenAvatar from '../TokenAvatar/index.ts';
...
+  const columns: ColumnDef<BalanceTableFieldModel, string>[] = useMemo(() => {
+    const columnHelper = createColumnHelper<BalanceTableFieldModel>();
 
-  const columns: ColumnDef<BalanceTableFieldModel, string>[] = useMemo(
-    () => [
+    return [
       columnHelper.display({
         id: 'asset',
         header: () => formatText({ id: 'table.row.asset' }),
+        headCellClassName: isMobile ? 'pr-2' : undefined,
```
Comment (bassgeta) [inline src/components/frame/v5/pages/BalancePage/partials/BalanceTable/hooks.tsx:85]: would it be possible to put these classNames inside of the `header` function? so we render a span with  this className or something

File: src/components/frame/v5/pages/BalancePage/partials/BalanceTable/hooks.tsx [hunk 2]
```diff
@@ -15,22 +20,70 @@ import TokenAvatar from '../TokenAvatar/index.ts';
...
+        cellContentWrapperClassName: isMobile ? 'pr-2' : undefined,
```
Comment (bassgeta) [inline src/components/frame/v5/pages/BalancePage/partials/BalanceTable/hooks.tsx:86]: would it be possible to put these classNames inside of the `cell` function? so we render a span with  this className or something

**Review by bassgeta — CHANGES_REQUESTED** [review]
Looks good, if my 2 comments are relevant, I think readability would be nicer, feel free to lemme know if it's not doable

**Review by melyndav — DISMISSED** [review]
Outside of Armando's feedback re: the lock icon, everything looks good. Please resolve this before merging.

**Comment by joanna-pagepro** [comment]
@ArmandoGraterol @melyndav  I can't make the textarea smaller so that the lock icon will be closer to the text. Also, on a design it looks the same as implemented: 
image

File: src/components/frame/v5/pages/BalancePage/partials/BalanceTable/hooks.tsx (hunk 1 above)
Comment (joanna-pagepro) [reply src/components/frame/v5/pages/BalancePage/partials/BalanceTable/hooks.tsx:85]: You can return a string only 
image
Also, the value that I want to change is set on `th` element, so a `span` wouldn't change anything in this case.

File: src/components/frame/v5/pages/BalancePage/partials/BalanceTable/hooks.tsx (hunk 2 above)
Comment (joanna-pagepro) [reply src/components/frame/v5/pages/BalancePage/partials/BalanceTable/hooks.tsx]: I can't find anything in the docs. I used a function that we already have implemented to update the spacings on the wrapper.

Comment (bassgeta) [reply src/components/frame/v5/pages/BalancePage/partials/BalanceTable/hooks.tsx]: can we try it out? it's probably not a huge thing, just add the className to the `cell` attribute like we already do in some cases
image

Comment (joanna-pagepro) [reply src/components/frame/v5/pages/BalancePage/partials/BalanceTable/hooks.tsx]: But this is an additional element inside the cell's wrapper, so it won't work the way I need it to work

**Review by bassgeta — APPROVED** [review]

**Review by iamsamgibbs — APPROVED** [review]
Looks good to me!

**Review by warrencolliver — APPROVED** [review]

**Review by ArmandoGraterol — APPROVED** [review]
I didn't see that screen! Looks good to me then 👍

**Pull request merged**